source/1_engine/rex_engine/include/rex_engine/diagnostics/win/win_call.h

```diff
@@ -12,20 +12,23 @@
 
 #pragma once
 
+#include "rex_engine/diagnostics/win/win_error.h"
+#include "rex_engine/diagnostics/win/win_types.h"
 #include "rex_std/bonus/string.h"
 #include "rex_std/bonus/types.h"
 
 namespace rex
 {
   namespace win
   {
-    using DWord = unsigned long;
+
+    REX_CREATE_STRONG_TYPE(ErrorSuccess, DWord);
 
     class WinCall
     {
     public:
-      WinCall(rsl::string_view file, rsl::string_view function, card32 lineNr);
-      WinCall(DWord errorSuccess, rsl::string_view file, rsl::string_view function, card32 lineNr);
+      WinCall(DWord funcResult, ErrorSuccess errorSuccess, rsl::string_view winFunc, rsl::string_view file, rsl::string_view function, card32 lineNr);
```

Comment:
because funcResult is the result of a function, initialized with the function call itself.
ErrorSuccess is there to make it a bit clearer and not have 2 integers next to each other which is error prone and would create a clang-tidy warning as well

Reply:
That still does not explain why it's not a STRONG TYPE, it can still be a STRONG TYPE if a function results fills it in and it can still be a STRONG TYPE if both funcResult and errorSuccess are STRONG TYPES.